net-misc/kea/kea-2.2.0.ebuild

```diff
@@ -10,51 +10,69 @@ MY_P="${PN}-${MY_PV}"
 DESCRIPTION="High-performance production grade DHCPv4 & DHCPv6 server"
 HOMEPAGE="https://www.isc.org/kea/"
 
-inherit autotools systemd tmpfiles
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit autotools fcaps python-single-r1 systemd tmpfiles
 
 if [[ ${PV} = 9999* ]] ; then
 	inherit git-r3
-	EGIT_REPO_URI="https://github.com/isc-projects/kea.git"
+	EGIT_REPO_URI="https://gitlab.isc.org/isc-projects/kea.git"
 else
 	SRC_URI="ftp://ftp.isc.org/isc/kea/${MY_P}.tar.gz
 		ftp://ftp.isc.org/isc/kea/${MY_PV}/${MY_P}.tar.gz"
-	[[ "${PV}" == *_beta* ]] || [[ "${PV}" == *_rc* ]] || \
-	KEYWORDS="~amd64 ~arm64 ~x86"
+	# odd minor version = development release
+	if [[ $(( $(ver_cut 2) % 2 )) -ne 1 ]] ; then
+		[[ "${PV}" == *_beta* ]] || [[ "${PV}" == *_rc* ]] || KEYWORDS="~amd64 ~arm64 ~x86"
```

Comment:
Just use a proper if statement instead and combine the ||s within one [[ ]].

Reply:
done